frontend/views.py

```diff
@@ -3,3 +3,7 @@
 
 def index(request):
     return render(request, 'frontend/index.html')
+
+
+def market(request):
+    return render(request, 'frontend/index.html')
```

Comment:
You should rather utilize the react router

Reply:
Done.